flake8_trio/visitors/flake8triovisitor.py

```diff
@@ -117,3 +118,12 @@ def save_state(self, node: ast.AST, *attrs: str, copy: bool = False):
     def walk(self, *body: ast.AST) -> Iterable[ast.AST]:
         for b in body:
             yield from ast.walk(b)
+
+    def visit_Import(self, node: ast.Import):
+        for alias in node.names:
+            name = alias.name
+            if name in ("trio", "anyio") and alias.asname is None:
+                if self.library is not None and name != self.library:
+                    self.library = f"[{self.library}|{name}]"
+                else:
+                    self.library = alias.name
```

Comment:
This is great, but I think we should also have a global `--anyio` flag to set the default library; otherwise `anyio` projects will have `trio` recommended to them in any file that doesn't happen to import `anyio`!